test/util/simulate-event.js

```diff
@@ -5,7 +5,17 @@
  * @returns {HTMLNode} The target of the event.
  */
 function simulateEvent( eventType, target, eventOption = {} ) {
-  const event = document.createEvent( 'Event', eventOption.currentTarget );
+  let event;
+
+  if ( eventType === 'click' ) {
+    event = new MouseEvent( 'click', {
+      bubbles: true,
+      cancelable: true,
+      view: window
+    } );
+  } else {
+    event = document.createEvent( 'Event', eventOption.currentTarget );
```

Comment:
This doesn't need to be changed to `window.document`, does it?

Reply:
Wouldn't hurt in the tests, since JSDOM is finicky